instrumentation/opentelemetry-annotations-1.0/README.md

```diff
@@ -0,0 +1,5 @@
+# Settings for the OpenTelemetry Annotations integration
+
+| Environment variable 	| Type 	| Default 	| Description 	|
+|-----------------	|------	|---------	|-------------	|
+| `TRACE_ANNOTATED_METHODS_EXCLUDE_CONFIG` | String |  | All methods to be excluded from auto-instrumentation by annotation-based advices. |
```

Comment:
For the sake of consistency, should document system property `otel.instrumentation.opentelemetry-annotations.exclude-methods`

Reply:
Added. But what's the relation between the `opentelemetry-annotations` instrumentation and the `external-annotations` one? How many settings do they have in common?

Reply:
One is exclusively for OpenTelemetry `WithSpan` annotation, another is for all 3rd party annotations with similar functionality that we support, e.g. NR or DD ones.